scripts/copyhooks.js

```diff
@@ -0,0 +1,12 @@
+const fs = require('fs');
+const path = require('path');
+
+const srcDir = './hooks/';
+const dstDir = './.git/hooks/'
```

Comment:
Will these be environment agnostic?
should it be something like
```suggestion
const srcDir = path.resolve(__dirname, 'hooks');
const dstDir = path.resolve(__dirname, '.git', 'hooks');
```
please verify...

Reply:
yes, they are.

Reply:
Tested on windows, mac and linux.

Reply:
Okay. Can be resolved.